### ISSUE TITLE
[Feature Request] : extreme mute

### ISSUE THREAD

**Issue by Rik44444** [telegramdesktop/tdesktop#7071]
Hi, i have a feature request:
an 'extreme mute' function, whereby you 1) do not receive notifications of the group or chat, but also extra for the extreme mute function **2) the group or chat also does not end up on the top of your list of messages when something is posted in that group or chat, but just stays either completely on the bottom or is ranked in your message list based on YOUR last post.** Use case: i have several groups that i cannot leave (eg some work groups) but where my input is not needed on a daily or even weekly basis. I'd like to be able to check this group every once in a while, when i feel like it, instead of being prompted every time that this group exists.

**Comment by MasterGroosha** [comment]
I think you can mute these groups and archive them. Muted chats don't leave archive on their own and you can even pin them there.

**Comment by Intenditore** [comment]
> 
> 
> I think you can mute these groups and archive them. Muted chats don't leave archive on their own and you can even pin them there.
> Read more about archived chats here: https://telegram.org/blog/archive-and-new-design

Chat archive works weirdly (it's stupid). It's always on top, it doesn't prevent archived chats/channels from showing up on the counter, it always disturbes with increasing "unread count". So, I wouldn't call it a solution

**Comment by MasterGroosha** [comment]
> It's always on top

You can either "hide" archived chats (that button will be twice as small) or even "move to main menu", completely removing Archive from chats list.

> it doesn't prevent archived chats/channels from showing up on the counter

You can exclude muted chats from unread counter (this was introduced years ago)

**Comment by RememberTheAir** [comment]
> It's always on top, it doesn't prevent archived chats/channels from showing up on the counter, it always disturbes with increasing "unread count". So, I wouldn't call it a solution

Neither of these were described as problems in the OP. Yes the archive is a possible solution to the problem described. You added a bunch of other issues that weren't in the initial description and declared the proposed solution as invalid because they're not addressed (which is partially not true) by the current implementation. Hmmm

**Comment by Intenditore** [comment]
> You can exclude muted (not only archived, but **all** muted) chats from unread counter (this was introduced years ago)

This is actually not a solution. What if there are chats I don't want to disturb me with their notifications, but I want to see them in Unread count??

**Comment by RememberTheAir** [comment]
@Intenditore then it belongs to another issue. The archive is a possible solution that meets what the opening post asked for. I believe there's already an issue about archived chats and unread counter. Will link it if I manage to find it

**Comment by Intenditore** [comment]
@RememberTheAir 
Here it is https://github.com/telegramdesktop/tdesktop/issues/6105

**Comment by Rik44444** [comment]
hi, i agree with @Intenditore  that the Archive is somewhat not-useful for my usecase. My endgoal is to not be disturbed anylonger by the 'extreme muted' chats. With the archive, this does not happen, because the Archive is always on top (for some strange reason) + it shows an unread counter, which triggers me every time. 
So it provides some solution, but definitely not all.

> 
> 
> > I think you can mute these groups and archive them. Muted chats don't leave archive on their own and you can even pin them there.
> > Read more about archived chats here: https://telegram.org/blog/archive-and-new-design
> 
> Chat archive works weirdly (it's stupid). It's always on top, it doesn't prevent archived chats/channels from showing up on the counter, it always disturbes with increasing "unread count". So, I wouldn't call it a solution

**Comment by MasterGroosha** [comment]
@Rik44444 
> because the Archive is always on top (for some strange reason) + it shows an unread counter, which triggers me every time.

As I said, you can move Archive to main menu, removing it from chats list.
![изображение](https://user-images.githubusercontent.com/2780044/74175004-0f84bb00-4c46-11ea-9e7b-c726cdd7323f.png)

**Comment by Rik44444** [comment]
ah brilliant! many thanks for this tip :)

**Comment by MasterGroosha** [comment]
@Rik44444 is your issue solved? If yes, please consider closing it.

**Comment by Rik44444** [comment]
well, not really, i'd still like an 'extreme mute' function, or more detailed custom muting options. Archiving is kind of a temporary fix.

**Comment by GlassedSilver** [comment]
> 
> 
> well, not really, i'd still like an 'extreme mute' function, or more detailed custom muting options. Archiving is kind of a temporary fix.

Pokemon Go player with a strong local community by any chance? :D 

We have a few bot channels here that keep you up to date on raids throughout the region... needless to say there's basically a new message every couple of seconds....

So that sucker is always on top below my pins which I kinda only use (the few I have) to move away the distraction those spammy (but still needed and useful) channels are. I just don't need them every day but when I need them I don't want to fuss about with going into the archive which I obviously hide because I like a clean UI.

I love the idea of extreme muting. If anything a "pin to bottom" would do the trick and could probably use the same function as the pin to top feature. Maybe without limits, because why does Telegram care how many pins I need? I digress...

**Comment by kmantel** [comment]
unstale
still interested in this

**Comment by zouden** [comment]
Yes this feature is sorely needed as Telegram becomes used for more than just chatting with friends.

**Comment by Intenditore** [comment]
Still no news :|
